react-checkout/src/App.css

```diff
@@ -41,3 +49,46 @@ input:-webkit-autofill {
   -webkit-box-shadow: 0 0 0 30px white inset;
   -webkit-text-fill-color: #212529 !important;
 }
+
+@media (min-width: 1200px) {
+  .card-fixed {
+    position: fixed;
+  }
+}
+
+pre {
+  white-space: pre-wrap; /* css-3 */
```

Comment:
When using vendor prefixes in CSS, the non-vendor prefixed version should always come last.

Reply:
Just moved this